### ISSUE TITLE
Workflow not responding

### ISSUE THREAD

**Issue by Benjamski** [vdesabou/alfred-spotify-mini-player#130]
This may be because of the Alfred 3 update but the worklow isn't responding. I can't open spotify fromt he workflow or do any of my usual actions like next track, pause/play or select playlists.

Should I reinstall? Or maybe reset the api connection somehow?

I'm happy to just go through the process again, I just wanted to see if there's any way to do it in a few less steps.

I'm not savvy with workflow development yet (though I really want to be) and I just noticed that when i view current track from the mini player it shows the current track. when I do this with the alfred debugger running and try to pause the current track I get the following output:

```py
Starting debug for 'Spotify Mini Player'

[2016-06-08 10:13:08][ERROR: action.script] /bin/bash: /Users/Benjamski/Library/Caches/com.runningwithcrayons.Alfred-3/Workflow Data/com.vdesabou.spotify.mini.player/action.log: No such file or directory
```

**Comment by vdesabou** [comment]
Hi,

Can you send by email the TGZ, see http://alfred-spotify-mini-player.com/articles/support/

Thanks

**Comment by vdesabou** [comment]
Actually, I think executing ```spot_mini_debug ``` should fix your problem. Let's see..

**Comment by Benjamski** [comment]
I tried running that but nothing happened. That should open my mail client with the debug info right?

It feels like there's a very loose connection somewhere. Like some core path has changed since it was last working.

**Comment by vdesabou** [comment]
Can you install this beta https://infinit.io/_/txeWG5J and try again?

**Comment by vdesabou** [comment]
Are you sure you're using version 6.3 ?

**Comment by Benjamski** [comment]
Sorry, I didn't see your message about the beta until just now.

So I'm not sure what was happening but here's what fixed it. I just uninstalled the workflow from alfred, reinstalled it from your site, set my hotkey back to what i use in Alfred, and it works perfectly, still authenticated and everything.

I'm really not sure what the issue was but it seemed like I was accessing all the right options, showing updates, showing current tracks, accessing my library and everything—but none of the actions fully executed. Maybe when I tried updating previously it wasn't actually updating? I don't know what caused that disconnect to begin with, apart from maybe my upgrade to Alfred 3.

I'll raise a new issue if I notice anything strange but everything's looking a-okay. Great work on this workflow, it's incredible. I'm using it as the pinnacle to work towards in learning how to build my own in the future.
